<launch>
        <node pkg="step_test" type="sm_traj_generator.py" name="generate_getup_poses_sm" output="screen" required="true"> 
            <param name="trajectory_dir"            value="$(find step_test)/scripts/getup/poses/front"/>
        </node>
        <include file="$(find cm730)/launch/cm730_no_torque.launch"/>
</launch>
